docs/content/latest/develop/ecosystem-integrations/_index.html

```diff
@@ -81,19 +81,31 @@
         <div class="title">Metabase</div>
       </div>
       <div class="body">
-        Ad-hoc interactive SQL queries on top of PostgreSQL-compatible YugabyteDB.
+        Ad-hoc interactive SQL queries on top of PostgreSQL-compatible YugabyteDB
       </div>
     </a>
   </div>
 
-   <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-     <a class="section-link icon-offset" href="spring/">
+  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
+    <a class="section-link icon-offset" href="spring/">
       <div class="head">
-        <img class="icon" src="/images/section_icons/develop/ecosystem/spring.png" aria-hidden="true" />      
+        <img class="icon" src="/images/section_icons/develop/ecosystem/spring.png" aria-hidden="true" />
       <div class="title">Spring Framework</div>
       </div>
       <div class="body">
-        Spring Framework support for YugabyteDB.
+        Spring Framework support for YugabyteDB
+      </div>
+    </a>
+  </div>
+
+  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
+    <a class="section-link icon-offset" href="databricks/">
+      <div class="head">
+        <img class="icon" src="/images/section_icons/develop/ecosystem/databricks.svg" aria-hidden="true" />
```

Comment:
Databricks logo is not getting displayed correctly in the preview. Probably needs an image of higher resolution.

Reply:
Odd, it's an SVG, so it's sharp no matter the resolution I try it at. Is the issue that it's too small?